### ISSUE TITLE
Malformed metric name

### ISSUE THREAD

**Issue by jbgi** [input-output-hk/cardano-byron-proxy#5]
Byron proxy expose a custom metric:
```
# TYPE int:byron-proxy_db_block count gauge
int:byron-proxy_db_block count  2876503.0
```
that is not recognized by prometheus:
`invalid metric type "proxy_db_block count gauge"`

I believe the `int:` prefix should be removed.

**Comment by jbgi** [comment]
Metrics is logged at https://github.com/input-output-hk/cardano-byron-proxy/blob/d61f01906e081283e150e4fcecc5505d023d99ce/src/exec/Main.hs#L349

**Comment by disassembler** [comment]
fixed by #76
